test-evidence/src/main/scala/com/daml/test/evidence/generator/Main.scala

```diff
@@ -22,26 +24,37 @@ object Main {
     println(s"Wrote to $path")
   }
 
+  private val workspaceRoot: Option[String] = sys.env.get("BUILD_WORKSPACE_DIRECTORY")
+  private val targetDir = workspaceRoot.map(File(_)).getOrElse(File.currentWorkingDirectory)
+
   private def writeEvidenceToCsvFile[TE <: TestEntryCsv](
       fileName: String,
       entries: List[TE],
   ): Unit = {
     println(s"Writing inventory to $fileName...")
-    val file = File(fileName)
+    val file = targetDir / fileName
```

Comment:
In the `println` statement, mention `file` instead of `fileName`, for improved clarity.